src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/ez-custom-style.js

```diff
@@ -0,0 +1,55 @@
+import EzConfigBase from './base';
+
+export default class EzCustomStyleConfig extends EzConfigBase {
+    constructor(customStyles = []) {
+        super();
+
+        this.name = 'custom-style';
+        this.buttons = [
+            'ezmoveup',
+            'ezmovedown',
+            this.getStyles(customStyles),
+            'ezblocktextalignleft',
+            'ezblocktextaligncenter',
+            'ezblocktextalignright',
+            'ezblocktextalignjustify',
+            'ezblockremove',
+        ];
+    }
+
+    getStyles(customStyles = []) {
+        return {
+            name: 'styles',
+            cfg: {
+                showRemoveStylesItem: false,
+                styles: [...customStyles],
```

Comment:
I made it work like this because when you change from a custom style to another, it doesn't clean the previous style information.

For example, considering the following style configuration :
`{ name: 'Style 1', element: 'div', attributes: { 'class': 'my_style' } }` 

Starting from
`<p>test<p>`
You select the style and it become
`<div class="my_style">test</div>`
Then if you re-select "paragraph" it become
`<p class="my_style">test</p>`

The only way to clear all the custom style configuration is to use the "remove style" button.